client/src/components/Dashboard/SchoolDashboard/queries.js

```diff
@@ -0,0 +1,52 @@
+import client, { gql } from '../../../clientQuery';
+
```

Comment:
Hm I think this file is part of joes PR as well. I think we may need to Merge that before this and check whether it conflicts

Reply:
Agreed - I'm behind master by 10 commits while ahead on README.md ones. I'll merge `master` with `readme` prior to the finishing the draft PR.